journalapp/views.py

```diff
@@ -1,6 +1,36 @@
-# from pyramid.response import Response
+# coding=utf-8
+from .models import DBSession, Entry
+from .form_new import NewBlogEntryForm
+from pyramid.httpexceptions import HTTPFound, HTTPNotFound
 from pyramid.view import view_config
-from journalapp.models import query_table, query_post
+from sqlalchemy import desc
+from sqlalchemy.exc import IntegrityError
+from webob.multidict import NoVars
+
+
+def query_table():
+    return DBSession.query(Entry).order_by(desc(Entry.created))
+
+
+def query_post(post_id):
+    return DBSession.query(Entry).get(post_id)
+
+
+def new_entry(new_title=None, new_text=None):
+    DBSession.add(Entry(title=new_title, text=new_text))
+    DBSession.flush()
+    new_id = DBSession.query(Entry).order_by(desc(Entry.created))[0].id
+    return new_id
```

Comment:
same here as well.  I like your style of breaking these down into external functions, but why not make them methods on a model class?